src/styles/utilits/normalise.scss

```diff
@@ -0,0 +1,27 @@
+html {
+  font-family: "Roboto", sans-serif;
+  line-height: normal;
+  font-style: normal;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+img,
+nav,
+div,
+a,
+p,
+.header,
+.catalog__cards {
+  box-sizing: border-box;
```

Comment:
Normillize is only for default browser styles, why do you reset them to your blocks and elements? Also, you set box-sizing: border-box; for all elements with * selector, you don't need to specify here.